src/test/java/org/broadinstitute/hellbender/tools/sv/cluster/CanonicalSVCollapserUnitTest.java

```diff
@@ -644,25 +644,7 @@ public Object[][] collapseSampleGenotypesTestData() {
                                 createGenotypeTestAttributes(2)
                         )
                 },
-                // het preferred over hom ref even with lower gq
```

Comment:
This should still be true right? I wasn't sure why this case got dropped, unless I didn't see one it's redundant with

Reply:
Good catch, I added this back in